collections/posts.coffee

```diff
@@ -1,9 +1,35 @@
-{ Collection, Model } = require 'backbone'
-{ BLOG_URL } = require('sharify').data
+contentful = require "contentful"
```

Comment:
Don't feel beholden to writing coffee script. In fact, I think we prefer if all new stuff is straight ES6 js!

Reply:
Actually, strike that. It matters little.